.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -89,6 +89,34 @@ steps:
           - github_commit_status:
               context: "agentbeat: Packaging"
 
+      - label: ":linux: Agentbeat packaging Linux FIPS"
+        key: "agentbeat-package-linux-fips"
+        depends_on:
+          - mandatory-win-2022-system-tests
```

Comment:
It's interesting that we are running `windows` tests here as part of DRA packaging for linux; but I didn't see the same set of tests available for linux, so likely makes sense to still test this functionality here?

Reply:
Agentbeat runs the mandatory system tests only on windows for some reason.
Then linux packaging happens and integration tests for linux are run.
I kept the same precedence between the steps for FIPS and the only previous step in the same group is the mandatory win tests. I think this is to avoid needless packaging if the system tests fail, if we want to package FIPS regardless of the system tests we can remove the requirement. 
Would that be ok, @pazone ?

Reply:
It turns out the agentbeat tests are not yet merged ([PR](https://github.com/elastic/beats/pull/41074)). Will figure out the ETA

Reply:
@pchila thanks for the details, I assumed something like this. Let's keep the tests as extra safety net for now.